spec/spec_helper.rb

```diff
@@ -72,12 +72,17 @@ def hash_inspect(hash)
 RSpec.shared_context "with modified configuration" do
   around do |example|
     configuration = example.metadata[:with_configuration]
-    key = configuration.keys.first
-    value = configuration.values.first
-    original = RSpec::Expectations.configuration.public_send(key)
-    RSpec::Expectations.configuration.public_send("#{key}=", value)
+    # Rework to use `slice` when we drop support for Ruby 2.3
```

Comment:
What would `slice` give us here, it just drops keys?

Reply:
Good catch. `RSpec::Expectations.configuration` isn't a Hash, so calling `slice` on it is impossible.
Removed this note.